src/v1.rs

```diff
@@ -381,6 +346,46 @@ impl SignatureAggregator {
     }
 }
 
+impl traits::Aggregator for Aggregator {
+    #[allow(non_snake_case)]
```

Comment:
Do you need this here?

Reply:
I got rid of pretty much all the `non_snake_case` variables and annotations.  I'm just leaving `R` since that's how schnorr proofs are commonly defined.